website/content/en/v0.5.5/getting-started-with-terraform/_index.md

```diff
@@ -311,10 +306,6 @@ spec:
       cpu: 1000
   provider:
     instanceProfile: KarpenterNodeInstanceProfile-${CLUSTER_NAME}
-    subnetSelector:
```

Comment:
These are now required, as of v0.5.5

Reply:
oh, let me put it back now, miseed it from 0.5.5 release notes :)